airflow/jobs/scheduler_job_runner.py

```diff
@@ -98,7 +98,8 @@
 DR = DagRun
 DM = DagModel
 
-RESCHEDULE_STUCK_IN_QUEUED_EVENT = "rescheduling stuck in queued"
+STUCK_IN_QUEUED_EVENT = "stuck in queued"
```

Comment:
one reason i removed the "rescheduling" part is because at the point where you log this, you don't know that it's reschedulable -- you only know that further down.